pywinauto/base_wrapper.py

```diff
@@ -499,17 +501,38 @@ def capture_as_image(self, rect=None):
                              "PIL is required for capture_as_image")
             return None
 
-        # get the control rectangle in a way that PIL likes it
         if rect:
-            box = (rect.left, rect.top, rect.right, rect.bottom)
+            control_rectangle = rect
+
+        # get the control rectangle in a way that PIL likes it
+        width = control_rectangle.width()
+        height = control_rectangle.height()
+        left = control_rectangle.left
+        right = control_rectangle.right
+        top = control_rectangle.top
+        bottom = control_rectangle.bottom
+        box = (left, top, right, bottom)
+
+        # check the number of monitors connected
+        if len(win32api.EnumDisplayMonitors()) > 1:
```

Comment:
`BaseWrapper` could be used on any platform, so I would recommend adding `sys.platform == "win32"` check before detecting second monitor.